### PR TITLE
feat: Design Event Invoice Template

### LINKED ISSUE #6197: Design Skeleton for Event Invoice Template
Parent Issue: #6123

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #6197 

#### Short description of what this resolves:
This prepares a skeleton template for serving event_invoice pdfs

#### Changes proposed in this pull request:

- Minor calculation changes in current cron job implementations
- Preparation of simple design of event invoice
- Modified buggy `calc_monthly_revenue` function

**Jinja Template Preview** : 

image

**PDF preview sample :** 
image

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by kushthedude** [comment]
@uds5501 Please include an `EventYay` logo on the top left margin

File: app/api/helpers/scheduled_jobs.py [hunk 1]
```diff
@@ -180,19 +180,28 @@ def send_monthly_event_invoice():
...
             currency = event.payment_currency
             ticket_fee_object = db.session.query(TicketFees).filter_by(currency=currency).one()
             ticket_fee_percentage = ticket_fee_object.service_fee
             ticket_fee_maximum = ticket_fee_object.maximum_fee
             orders = Order.query.filter_by(event=event).all()
             gross_revenue = event.calc_monthly_revenue()
-            ticket_fees = event.tickets_sold * ticket_fee_percentage
-            if ticket_fees > ticket_fee_maximum:
-                ticket_fees = ticket_fee_maximum
+            ticket_fees = gross_revenue * (ticket_fee_percentage / 100)
```
Comment (mrsaicharan1) [inline app/api/helpers/scheduled_jobs.py]: I think this should be `event.tickets_sold *  (ticket_fee_percentage / 100) `

File: app/api/helpers/scheduled_jobs.py [hunk 2]
```diff
@@ -180,19 +180,28 @@ def send_monthly_event_invoice():
...
+            if ticket_fees > ticket_fee_maximum * event.tickets_sold:
```
Comment (mrsaicharan1) [inline app/api/helpers/scheduled_jobs.py]: `ticket_fees > ticket_fee_maximum`

**Comment by uds5501** [comment]
@kushthedude I am trying to keep the basic design as minimal as possible. This can be improved with discussions and follow up iterations

File: app/api/helpers/scheduled_jobs.py (hunk 2 above)
Comment (uds5501) [reply app/api/helpers/scheduled_jobs.py]: @mrsaicharan1 completed the changes.

**Comment by kushthedude** [comment]
> @kushthedude I am trying to keep the basic design as minimal as possible. This can be improved with discussions and follow up iterations

Its good to go then.

**Comment by uds5501** [comment]
@iamareebjamal please review.

**Comment by iamareebjamal** [comment]
I'm on hold for any feature to ensure there is no critical bug fix needed, so that we can have a hotfix. There has been many bugs in recent releases due to lack of testing

**Pull request merged**